### ISSUE TITLE
BetterAnimations Could not be compiled

### ISSUE THREAD

**Issue by TheGreatTaz** [arg0NNY/DiscordPlugins#90]
### Plugin:

🎞️ BetterAnimations

### What version of BetterDiscord are you using?

1.10.2

### What version of the selected plugin(s) are you using (if applicable)?

1.1.19

### What client are you using?

🔵 Stable

### Describe the bug

An error was encountered while trying to load this plugin.

### Additional comments

```
TypeError: Cannot destructure property 'Route' of 'WebpackModules.getByProps(...)' as it is undefined.
    at plugin (betterdiscord://plugins/BetterAnimations.plugin.js:126:28)
    at eval (betterdiscord://plugins/BetterAnimations.plugin.js:2010:16)
    at eval (betterdiscord://plugins/BetterAnimations.plugin.js:2011:7)
    at eval (betterdiscord://plugins/BetterAnimations.plugin.js:2012:3)
    at N.requireAddon (betterdiscord/renderer.js:5:28601)
    at N.loadAddon (betterdiscord/renderer.js:5:6221)
    at N.loadAddon (betterdiscord/renderer.js:5:26269)
    at N.loadAllAddons (betterdiscord/renderer.js:5:9278)
    at N.initialize (betterdiscord/renderer.js:5:3093)
    at N.initialize (betterdiscord/renderer.js:5:25382)
    at Object.startup (betterdiscord/renderer.js:5:408589)
```

### I have read and agree to the following:

- [X] I have searched existing/closed issues to confirm that this hasn't been solved already.
- [X] I've already tried to disable all other plugins (or themes) to confirm this issue is related to the plugin.
This isn't necessarily required

**Comment by arg0NNY** [comment]
Fixed in 2df4bfa866fe105c93e6c19d3ba3a009cedfaed8

**Comment by AshSlake** [comment]
deu certo , vlw @arg0NNY  👍🏼 :)
